python3/ltk/managers.py

```diff
@@ -153,6 +153,13 @@ def get_doc_ids(self):
             doc_ids.append(entry['id'])
         return doc_ids
 
+    def get_process_ids(self):
+        """ returns all the process ids of documents that the user has added"""
+        process_ids = []
+        for entry in self._db.all():
+            process_ids.append(entry['process_id'])
```

Comment:
Account for cases where there is no process id.  When this gets released, it will updated instances where users already have documents in local tracking, and their databases won't have that key for documents added before the release.

Reply:
Check for process id key has been added